tools/templates/vfvs_run.py

```diff
@@ -412,7 +443,9 @@ def submit_ligand_for_docking(ctx, docking_queue, ligand_name, ligand_path, coll
                 'threads_per_docking': int(ctx['main_config']['threads_per_docking']),
                 'temp_dir': temp_dir,
                 'attrs': ligand_attrs,
-                'output_dir': str(ligand_directory_directory)
+                'output_dir': str(ligand_directory_directory),
+                'run_pose_check': int(ctx['main_config']['run_pose_check']),
+                'pose_check_timeout': int(ctx['main_config']['pose_check_timeout'])
```

Comment:
If someone uses an old all.ctrl file, this will crash during the run. I'd suggest updating the vfvs_prepare_folders.py to check for empty and set a default (off) if not in the file. Look at line 255 for an example of when sensor_screen_mode was added.